tests/fixtures/Requests/request-send-test.yml

```diff
@@ -0,0 +1,151 @@
+                          </productRequest>
+                          <productRequest formatCode="NA" statusCode="ENV" trackChanges="false">
+                              <language code="BG"/>
+                          </productRequest>
+                          <productRequest requestedDeadline="2019-04-15T12:00:00+02:00" formatCode="NA" statusCode="ENV" trackChanges="false">
+                              <language code="CS"/>
+                          </productRequest>
+                          <productRequest requestedDeadline="2019-11-15T12:00:00+01:00" formatCode="NA" statusCode="ENV" trackChanges="false">
+                              <language code="DE"/>
+                          </productRequest>
+                      </productRequests>
+                      <auxiliaryDocuments>
+                          <auxiliaryDocument format="TXT" type="REF" name="test1.txt" language="ML"/>
+                          <auxiliaryDocument format="TXT" type="REF" name="test1.txt" language="RO"/>
+                      </auxiliaryDocuments>
+                  </return>
+              </ns0:createRequestsResponse>
+          </S:Body>
+      </S:Envelope>
+    request:
+      linguisticRequest:
+        - contacts:
+            contact:
+              - UserId: '1'
+                RoleCode: AUTHOR
+              - UserId: '2'
+                RoleCode: EDITOR
+          generalInfo:
+            Title: Test
+            InternalReference: '1'
+            InternalTechnicalId: '1'
+            RequestedDeadline:
+              time: '2020-02-02'
+              timezone: null
+            Sensitive: false
+            DocumentToBeAdopted: true
+            DecideReference: decideReference
+            SentViaRUE: true
+            DestinationCode: PUBLIC
+            SlaAnnex: ''
+            SlaCommitment: ''
+            Comment: ''
+            OnBehalfOf: ''
+            AccessibleTo: ''
+  expectations:
+    request: |
+      <?xml version="1.0" encoding="UTF-8"?>
+      <SOAP-ENV:Envelope xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/" xmlns:ns1="http://eu.europa.ec.dgt.epoetry">
+      <SOAP-ENV:Body>
+      <ns1:createRequests>
+      <linguisticRequest><generalInfo><title>Test</title><internalReference>1</internalReference><internalTechnicalId>1</internalTechnicalId><requestedDeadline>2020-02-02T00:00:00+00:00</requestedDeadline><sensitive>false</sensitive><documentToBeAdopted>true</documentToBeAdopted><decideReference>decideReference</decideReference><sentViaRUE>true</sentViaRUE><destinationCode>PUBLIC</destinationCode><slaAnnex></slaAnnex><slaCommitment></slaCommitment><comment></comment><onBehalfOf></onBehalfOf><accessibleTo></accessibleTo></generalInfo>
+      <contacts><contact userId="1" roleCode="AUTHOR"/><contact userId="2" roleCode="EDITOR"/></contacts>
+      </linguisticRequest>
+      </ns1:createRequests>
+      </SOAP-ENV:Body>
+      </SOAP-ENV:Envelope>
+    assertions:
```

Comment:
For me it's weird to see assertions inside the fixtures. I think these should live in the tests file. Fixtures are something that we provide to mock data and to compare against. So for me we shouldn't have the tests here.
I also have doubts about the expectations.

Reply:
This is the whole point of this PR... if using expression language (_and thus, assertions in the yaml_), then we had to have this discussion before doing the PR  :(

Reply:
Assertions and fixtures goes together, you assert against a fixture, this is used everywhere, in data providers you pass the data of the test and the expected result so that you can use the same test to test more than one scenario.

Look at [data provider doc](https://phpunit.de/manual/3.7/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers) to have an example of it:

```php
<?php
class DataTest extends PHPUnit_Framework_TestCase
{
    /**
     * @dataProvider provider
     */
    public function testAdd($a, $b, $c)
    {
        $this->assertEquals($c, $a + $b);
    }

    public function provider()
    {
        return array(
          array(0, 0, 0),
          array(0, 1, 1),
          array(1, 0, 1),
          array(1, 1, 3)
        );
    }
}
?>
```

Reply:
@ademarco I never saw the tests being added in fixtures (while fixtures being added in tests like the example you're showing is common.
But ok thanks for clarifying.